PorticoExportPlugin.php

```diff
@@ -54,13 +58,13 @@ public function display($args, $request)
                 }
                 try {
                     // create zip file
-                    $path = $this->_createFile($issueIds);
+                    $path = $this->createFile($issueIds);
```

Comment:
I haven't seen how the `JATS template plugin` is working, but I know it's expected to generate the same type of content, so if you have enough time, it's an opportunity to drop the `PorticoExportDom.php`.